tests/unit/test_cli.py

```diff
@@ -155,3 +156,30 @@ def test_cli_output_path(create_tmp_simulation_config_file):
         f"Directory '{simconfig_output_path}' should NOT exist."
     )
     assert (tmp_path / new_output).is_dir(), f"Directory '{new_output}' not found."
+
+
+@pytest.mark.parametrize(
+    "create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "target_simulator": "CORENEURON",
+            "reports": {
+                "soma_v": {
+                    "type": "compartment",
+                    "variable_name": "v",
+                    "sections": "soma",
+                    "dt": 0.1,
+                    "start_time": 0.0,
+                    "end_time": 40.0
+                }
+           },
+       }
+    }
+], indirect=True)
+def test_cli_report_buff_size(create_tmp_simulation_config_file):
+    command = ["neurodamus", create_tmp_simulation_config_file, "--report-buffer-size=64", "--keep-build"]
+    subprocess.run(command, check=True, capture_output=True)
+    
+    report_confs = utils.ReportConf.load("build/report.conf")
+    assert report_confs.reports["soma_v.h5"].buffer_size == 64
```

Comment:
we are not testing for:

```
Neurodamus(create_tmp_simulation_config_file, report_buffer_size=64)
```

for example. Other commands got their own test file (like direct mode and save restore). Maybe this one is simple enough that can stay in here... or add a new file. As you prefer

In addition, we could test at least the cli a little more. What happens if:
- buffer size is a prime number? like 11
- buffer size is very big? is everything ok? We never dump data to file in that case?
- is 0? We should check that, indeed, the error is raised
- is negative? same as before
- is empty?
- is not an int/number

Reply:
yes, that was mainly my point

Reply:
Added test to confirm error handling of negative and 0 values